float32 x_pixel_diff
float32 y_pixel_diff
---
float32 steering_correction
